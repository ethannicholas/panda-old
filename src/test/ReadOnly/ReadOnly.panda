class ReadOnly {
    @readonly
    var test:Int
}

class Test {
    @class
    method main() {
        var r := new ReadOnly()
        r.test := 5
        Console.writeLine(r.test)
    }
}