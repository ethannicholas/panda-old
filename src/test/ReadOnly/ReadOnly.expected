error: ReadOnly.panda:10:10: 'ReadOnly.test' may not be modified
1 error
